tools/nsp_scripts/train_model.py

```diff
@@ -9,6 +9,7 @@
 import os
```

Comment:
should remove all the changes in this file

Reply:
same as above